doc/introduction/measurements.rst

```diff
@@ -178,8 +201,8 @@ a tensor of tensors is returned to provide differentiability for the outputs of
     def circuit():
         qml.Hadamard(wires=0)
         qml.CNOT(wires=[0,1])
-        qml.PauliX(wires=2)
-        return qml.expval(qml.PauliZ(0)),qml.expval(qml.PauliZ(1)), qml.sample(counts=True)
+        qml.PauliX(wires=1)
```

Comment:
Remove third qubit of the output of `print(result)`:

```pycon
[tensor(0.026, requires_grad=True) tensor(0.026, requires_grad=True)
 tensor({'01': 513, '11': 487}, dtype=object, requires_grad=True)]
```

Moreover, I get a tuple instead of a tensor as output. @antalszava was this changed in the return types PR?